File: medolylibrary/src/main/java/com/wa2c/android/medoly/library/PluginAction.java
package com.wa2c.android.medoly.library;

/**
 * Plugin action.
 */
public enum PluginAction {
    /** Media action plugin. */
    ACTION_MEDIA,
    /** Search action plugin. */
    ACTION_SEARCH;



    /** Constructor. */
    PluginAction() {
        actionValue = "com.wa2c.android.medoly.plugin.action." + this.name();
    }

    /** Action value. */
    private String actionValue;
    /** Get the action value. */
    public String getActionValue() {
        return actionValue;
    }

}

File: medolylibrary/src/main/java/com/wa2c/android/medoly/library/PluginAction.kt
package com.wa2c.android.medoly.library

/**
 * Plugin action.
 */
enum class PluginAction {
    /** Media action plugin.  */
    ACTION_MEDIA,
    /** Search action plugin.  */
    ACTION_SEARCH;



    /** Action value.  */
    val actionValue: String = "com.wa2c.android.medoly.plugin.action." + this.name
}
